src/commands/InitCommand.ts

```diff
@@ -81,11 +93,22 @@ export class InitCommand implements yargs.CommandModule {
     // Protected Static Methods
     // -------------------------------------------------------------------------
 
+    protected static executeCommand(command: string) {
+        return new Promise<string>((ok, fail) => {
+            exec(command, (error: any, stdout: any, stderr: any) => {
+                if (stdout) return ok(stdout);
+                if (stderr) return ok(stderr);
```

Comment:
Wouldn't `stderr` then a failure exit code cause this to not reject?  Because it would resolve first and then never reject.

Reply:
Hmm.. I took this implementation from `VersionCommand.ts`. Should I call fail for `stderr`?

Reply:
Yes, I think `fail` should be called here. Also, probably `VersionCommand` is incorrect.

Reply:
done